deploy/local/live-csc/nginx.conf

```diff
@@ -1,14 +1,16 @@
 upstream manager_load_balancer {
-    server love-manager-mount-01:8000;
-    server love-manager-mount-02:8000;
+    server localhost:8000;
+    #server love-manager-mount-01:8000;
```

Comment:
And again. In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.

Reply:
Oh sorry about this, that was not intended to be pushed. I will remove this part.